code/_onclick/hud/fullscreen.dm

```diff
@@ -45,17 +43,38 @@
 		clear_fullscreen(category)
 
 /datum/hud/proc/reload_fullscreen()
-	var/list/screens = mymob.screens
-	for(var/category in screens)
-		mymob.client.screen |= screens[category]
+	if(mymob.client)
+		var/obj/screen/fullscreen/screen
+		var/list/screens = mymob.screens
```

Comment:
not saying you have to change it, but interesting they don't use `mymob`.

Reply:
Aye, they use the viewmob in order to support their system where observing someone through OOC lets you see their HUD. We don't have that yet, so no need for that one.

Reply:
Right-o.